deploy/helm/flightctl/templates/flightctl-default-user.yaml

```
@@ -0,0 +1,47 @@
+    app: flightctl
+    component: default-user
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: org-default
+subjects:
+  - kind: ServiceAccount
+    name: flightctl-admin
+    namespace: {{ default .Release.Namespace .Values.global.auth.k8s.rbacNs }}
+{{- end }}
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)